navigator/backend/src/main/scala/com/digitalasset/navigator/data/DatabaseActions.scala

```diff
@@ -38,7 +38,7 @@ class DatabaseActions extends LazyLogging {
     */
   private val xa = Transactor.fromConnection[IO](
     DriverManager.getConnection("jdbc:sqlite::memory:"),
-    ExecutionContext.global)
+    Blocker liftExecutionContext ExecutionContext.global)
```

Comment:
As mentioned on typelevel/cats-effect#556,

> This must not be used with general purpose contexts like `scala.concurrent.ExecutionContext.Implicits.global`.

More details in the 71e59e1b3ef8ae66fb79f93cd94f71af8689e736 message

> The point of using Blocker instead of ExecutionContext seems to be to especially emphasize to API users that it isn't appropriate to use an ExecutionContext with ordinary behavior.